blocks/bricks/__init__.py

```diff
@@ -570,6 +570,44 @@ def apply_outputs(self):
         return self.application_methods[-1].outputs
 
 
+class Concatenate(Brick):
+    """Merges several variables by concatenating them.
+
+    Parameters
+    ----------
+    axis : int
+        Axis along which to concatenate inputs.
+
+    Notes
+    -----
+    This is a very straightforward wrapping of a Theano operation without
+    parameters of its own. It's mainly useful in conjunction with the
+    :class:`~blocks.filter.VariableFilter`.
```

Comment:
To be honest I am afraid to create a precedent. Theano has lots of ops, and if we have a wrapper in Blocks for each and every one of them, this will hugely increase the codebase. 

What about 

```python 
class Expression(Brick): # choose a better name
   """Wraps a Theano expression in a brick."""
   def __init__(self, expression, keyword_arguments, **kwargs):
      super(Expression, self).__init__(**kwargs)
      self.expression = expression
      self.keyword_arguments = keyword_arguments

   def apply(self, *args, **kwargs):
      return self.expression(*args, **dict_union(kwargs, keyword_arguments))

concatenate = Expression(tensor.concatenate, axis=1)
```

?